tests/internal/utils.c

```diff
@@ -37,6 +37,88 @@ struct url_check url_checks[] = {
     {-1, "://", NULL, NULL, NULL, NULL},
 };
 
+void test_url_split_sds()
```

Comment:
Looks like this test isn't in the list of tests to be run, at the end of this file.

Reply:
Good catch, fixed in new commit